arm-network/2017-06-01/examples/SubnetCreate.json

```diff
@@ -0,0 +1,32 @@
+{
+  "parameters": {

```

Comment:
Updated, please have a look. I think there should be no "name" property under "subnetParameters" as it is placed one level above